examples/cors/apicast-cors.yml

```diff
@@ -0,0 +1,231 @@
+apiVersion: v1
+kind: Template
```

Comment:
Well, there's also a chance that paths or implementation of the customized function will change in future... So another idea I had was to point to `apicast:v3.0.0` instead of `:master` to make sure it doesn't break.

I am not familiar with oc patch... Will check it out. Feel free to commit it too :)